packages/main/src/Avatar.js

```diff
@@ -82,6 +96,29 @@ const metadata = {
 			type: String,
 			defaultValue: AvatarSize.S,
 		},
+
+		/**
+		 * Defines the fit type of the desired image.
+		 * @type {String}
+		 * @defaultvalue "Cover"
+		 * @public
+		 */
+		imageFitType: {
+			type: String,
+			defaultValue: AvatarFitType.Cover,
+		},
+
+		/**
+		 * Defines the background color of the desired image.
+		 * @type {String}
+		 * @defaultvalue "Accent6"
```

Comment:
add info for the available options are: see the "size" property docs for reference.

Reply:
Done.